colorfield/templates/colorfield/color.html

```diff
@@ -1,9 +1,11 @@
 <input type="text"
  id="{{ id }}"
- class="colorfield_field jscolor {% if class %}{{ class }}{% else %}form-control{% endif %}"
+ class="colorfield_field coloris {{ id }} {% if class %}{{ class }}{% else %}form-control{% endif %}"
  name="{{ name }}"
  value="{% firstof value default '' %}"
  placeholder="{% firstof value default '' %}"
- data-jscolor="{hash:true,width:225,height:150,format:'{{ format }}',required:{% if required %}true{% else %}false{% endif %},paletteCols:4,paletteHeight:28{% if palette %},palette:{{ palette|safe }}{% endif %}}"
+ data-coloris="{{ data_coloris_id }}"
```

Comment:
Since `data-coloris` attribute is used by `coloris` lib and we don't know how it will be used in the future, I would use another data attribute for passing the script id, eg. `data-coloris-options-json-script-id` (very explicit name and no room for doubts).